coding-challenges/README.md

```diff
@@ -111,3 +111,54 @@ Searching algorithm should correctly return the blogs that contain the search qu
 Test cases should cover basic scenarios and edge cases for the searching algorithm.
 
 ```
+
+3.Implement a Basic Linux Command Line Interface (CLI) Tool
+
+```bash
+**Context**
+
+To broaden the scope of learning and provide hands-on experience with command line tools, we want to introduce a challenge focused on building a basic Linux Command Line Interface (CLI) tool. This will allow beginners to get familiar with common Linux commands and to provide as a starting point to working with the terminal.
+
+**The Project**
+
+You are tasked with developing a simple CLI tool that performs basic file management tasks. Users should be able to interact with the tool via the command line to perform actions such as creating files, listing directory contents, renaming files, and deleting files.
+
+**Requirements**
+
+1. **CLI Commands**:
+    - Implement commands for basic file management tasks, including:
+    - `create`: Create a new file.
+    - `list`: List the contents of a 
+    - `rename`: Rename a file.
+    - `delete`: Delete a file.
+
+2. **Command Syntax**:
+   - Define a clear and intuitive syntax for each command, along with any required arguments or options.
+
+3. **File Management**:
+   - Ensure that file management operations are performed safely and accurately.
+
+4. **Error Handling**:
+   - Implement proper error handling to provide informative error messages to users.
+
+5. **Documentation**:
+   - Provide clear documentation within the CLI tool itself or in a separate README file explaining how to use each command and any available options.
```

Comment:
For the documentation I would suggest to recommend creating a command `help` to provide details about the different commands